open_mastr/xml_download/utils_write_to_database.py

```diff
@@ -154,80 +149,26 @@ def correct_ordering_of_filelist(files_list: list) -> list:
     return files_list
 
 
-def preprocess_table_for_writing_to_database(
-    f: ZipFile,
-    file_name: str,
-    xml_tablename: str,
-    bulk_download_date: str,
-) -> pd.DataFrame:
-    data = f.read(file_name)
-    try:
-        df = pd.read_xml(data, encoding="UTF-16", compression="zip")
-    except lxml.etree.XMLSyntaxError as err:
-        df = handle_xml_syntax_error(data.decode("utf-16"), err)
-
-    df = add_zero_as_first_character_for_too_short_string(df)
-    df = change_column_names_to_orm_format(df, xml_tablename)
-
-    # Add Column that refers to the source of the data
-    df["DatenQuelle"] = "bulk"
-    df["DatumDownload"] = bulk_download_date
-    return df
+def read_xml_file(f: ZipFile, file_name: str) -> pd.DataFrame:
+    """Read the xml file from the zip file and return it as a DataFrame."""
+    with f.open(file_name) as xml_file:
+        try:
+            return pd.read_xml(xml_file, encoding="UTF-16", parser='etree')
```

Comment:
We use black as a code formatter, so that code style is the same everywhere in the repository. Can you install the pre-commit hooks as described [here](https://github.com/OpenEnergyPlatform/open-MaStR/blob/production/CONTRIBUTING.md#23-commit-your-changes) or directly install the code formatter [black](https://black.readthedocs.io/en/stable/). When running black on this file, it would do a few changes (like changing the 'etree' to "etree").

Reply:
Sure, I will do that! Thanks for catching these inconsistencies